packages/components/src/components/wizard/wizard.tsx

```diff
@@ -15,28 +31,26 @@ const Wizard = ({ children, className, initial_step, onStepChange, nav, selected
         }
     }, [initial_step, getSteps]);
 
-    const onChangeStep = stats => {
+    const onChangeStep = (stats: { [key: string]: number }) => {
         // User callback
-        onStepChange(stats);
+        if (typeof onStepChange === 'function') onStepChange(stats);
```

Comment:
maybe it would be ok just to use this:
onStepChange?.(stats)

Reply:
Will change this as suggested